pvlib/tests/test_modelchain.py

```diff
@@ -918,6 +918,18 @@ def test_run_model_from_effective_irradiance(sapm_dc_snl_ac_system, location,
     assert_series_equal(ac, expected)
 
 
+def test_run_model_from_effective_irradiance_no_poa_global(
```

Comment:
Do we also need a test for when both effective irradiance and poa global are provided? I think not but thought it was worth mentioning.

Reply:
I think it's worth adding this test. All of the tests for `run_model_from_effective_irradiance` copy the values for 'poa_global' to 'effective_irradiance' (for convenience) so can't distinguish if the wrong values are being used to calculate power.